charts/turing/README.md

```diff
@@ -54,6 +54,13 @@ $ helm uninstall turing
 The command removes all the Kubernetes components associated with the chart and deletes the release.
 This includes the dependencies that were installed by the chart. Note that, any PVCs created by the chart will have to be deleted manually.
 
+### Rendered field
+* The purpose of `.Values.rendered.releasedVersion` is to update all configs that share this tag using this 1 value in values.yaml
+`.Values.rendered.releasedVersion` is used in rendering `merlin.config` partial template and `merlin.deploymentTag` partial template
+If .Values.deployment.image.tag is specified, it will overwrite the value in .Values.releasedVersion
+Values.  Rendered values will overwrite values in `.Values.config`
+Default value for .Value.deployment.image.tag and .Values.config is set to empty string respectively
```

Comment:
Same comment here as for the Merlin readme, the formatting of this paragraph is a bit weird ` .Values.deployment.image.tag` and `.Values.config` aren't in backticks, and `Values.  Rendered values will overwrite values in `.Values.config``  looks weird 😅 Was this paragraph generated automatically? 😮

Reply:
Yep will fix this

Reply:
Fixed